android/src/main/java/com/revenuecat/purchases_flutter/PurchasesFlutterPlugin.java

```diff
@@ -71,6 +71,7 @@ public void onMethodCall(MethodCall call, Result result) {
                 String apiKey = call.argument("apiKey");
                 String appUserId = call.argument("appUserId");
                 Boolean observerMode = call.argument("observerMode");
+                String userDefaultsSuiteName = call.argument("userDefaultsSuiteName"); // iOS-only, unused.
```

Comment:
we could also just not read this value at all, but I figured having something explicit might be better to that future devs wouldn't be confused by the disparity between iOS and Android implementations